### PR TITLE
Update Upgrading.md

### PR DESCRIPTION
**Pull request by jim-slattery-rs**
Tiny typos.

### CODE REVIEW

File: docs/Upgrading.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ var Person = MediaType("application/vnd.goa.person", func() {
 		Attribute("age", Integer, "Age of person")
 		Required("name")
 	})
-	View("default", func() { // View defines a rendering of the media type.
+	View("default", func() {  // View defines a rendering of the media type.
 		Attribute("id")   // Media types may have multiple views and must
 		Attribute("href") // have a "default" view.
```
Comment (jim-slattery-rs) [inline docs/Upgrading.md]: should these actually just be `name` and `age`? Not sure where `id` and `href` come from.

Comment (nitinmohan87) [reply docs/Upgrading.md]: It must be `name` and `age`. Good catch!

Comment (jim-slattery-rs) [reply docs/Upgrading.md]: Fixed 👍

**Comment by raphael** [comment]
Whoops, thank you!

**Pull request merged**